### ISSUE TITLE
Anchor inside of an anchor - gulp error

### ISSUE THREAD

**Issue by ghost** [vuejs/vue-template-validator#2]
hi,
I am using **Laravel**, **gulp**, **browserify** and **vueify** . When I try to combine everything (**gulp** || **gulp watch**) I get an error. I noticed that I only get an error when there is an anchor inside of another anchor.

```
<a href="javascript;">
    <a href="javascript;"></a>
</a>
```

The error says:

```
node_modules\vue-template-validator\lib\rules\self-closing-tag.js:26
    node.__location.startTag &&
                   ^

TypeError: Cannot read property 'startTag' of undefined
    at module.exports (E:\Laravel\ProjectTarsands\node_modules\vue-template-validator\lib\rules\self-closing-tag.js:26:20)
    at E:\Laravel\ProjectTarsands\node_modules\vue-template-validator\index.js:21:5
    at Array.forEach (native)
    at validate (E:\Laravel\ProjectTarsands\node_modules\vue-template-validator\index.js:20:9)
    at E:\Laravel\ProjectTarsands\node_modules\vue-template-validator\index.js:25:7
    at Array.forEach (native)
    at validate (E:\Laravel\ProjectTarsands\node_modules\vue-template-validator\index.js:24:21)
    at E:\Laravel\ProjectTarsands\node_modules\vue-template-validator\index.js:25:7
    at Array.forEach (native)
    at validate (E:\Laravel\ProjectTarsands\node_modules\vue-template-validator\index.js:24:21)
```

Actual code that produced a bug:

```
<template>
    <div class="BasicList">
        <a href="javascript">
            <div class="box" style="height: 300px; width: 300px;">
                <a href="javascript;">LINK</a>
            </div>
        </a>
    </div>
</template>

<script>
</script>

<style lang="stylus">
    @require './resources/assets/stylus/colors.styl'
    @require './resources/assets/stylus/metrics.styl'   

    .BasicList {
    }

</style>
```

The code **crashes** both **gulp** and **gulp watch** .

I am not sure if this is a bug or a feature, since it seems browsers do not allow it anyway - http://stackoverflow.com/a/13054959/2674529 .

If nothing else there could be a description of the problem, not just self-closing-tag.js ...

**Comment by yyx990803** [comment]
Can you check the version of `vue-template-validator` in your npm dependencies? I believe this should've already been fixed in the latest version.

**Comment by ghost** [comment]
It is fixed in the newest version (1.1.4) :+1: 

I was using v1.0.1.. sorry for the hassle.
